.github/workflows/ci.yaml

```diff
@@ -80,7 +80,7 @@ jobs:
             ccache-${{ env.CACHE_PREFIX }}
       # https://github.com/ros-industrial/industrial_ci/pull/649
       - name: industrial_ci
-        uses: 'tylerjw/industrial_ci@clang-tidy-modified-filter'
+        uses: 'ros-industrial/industrial_ci@master'
```

Comment:
nice, could you also remove the PR link above?

Reply:
Yes, I'll make a separate PR for just this change too.